controller/user.controller.js

```diff
@@ -36,19 +13,24 @@ module.exports = {
         if(!checkParamValidity(userName)){
             
             return res.status(400).json({
-                success: 0,
+                success: false,
                 message: "invalid request"
             });
         
         }
         
         try{
-            const results = await getUserByUsername(userName);
+            const results = await getUserByUsername(convertToLowerCase(userName));
```

Comment:
```suggestion
            const result = await getUserByUsername(convertToLowerCase(userName));
```
the method indicates that it will return only one user but the variable name is plural